garden/src/features/gardens/components/AssociatedMaterials.tsx

```diff
@@ -23,14 +23,14 @@ const AssociatedMaterials = ({ resource }: { resource: Entrypoint | ModalFunctio
         <AccordionContent className="grid grid-cols-1 gap-4 py-6 lg:grid-cols-2">
           <>
             {resource.papers?.map((paper) => (
-              <div key={paper.doi} className="rounded-md border bg-white p-6">
+              <div key={paper.doi || paper.url} className="rounded-md border bg-white p-6">
                 <div className="mb-4 flex items-center justify-between gap-4">
                   <div className="grid grid-cols-12 items-center gap-2">
                     <BookOpen className="col-span-1 hidden h-6 w-6 text-gray-600 sm:block" />
 
                     <div className="col-span-11">
                       <Link
-                        to={`https://doi.org/${paper.doi}`}
```

Comment:
Does this mean that if the paper doesn't have a url or a DOI, we have a dummy link that does nothing?

Reply:
yes but the dummy link does not seem to be clickable so I don't think it really matters -- that said I'm still interested in a more idiomatic way of doing this if there's an obvious one just for my own sake